packages/html-reporter/src/testResultView.tsx

```diff
@@ -14,7 +14,7 @@
   limitations under the License.
 */
 
-import type { TestAttachment, TestCase, TestResult, TestStep } from './types';
+import type { ErrorDetails, TestAttachment, TestCase, TestResult, TestStep } from './types';
```

Comment:
You mentioned you wanted to html reporter separately, let's only plumb the error fields for now.